src/ergw_aaa_radius.erl

```diff
@@ -171,10 +171,21 @@ validate_ip(_Opt, {_,_,_,_,_,_,_,_} = IP) ->
 validate_ip(Opt, IP) ->
     throw({error, {options, {Opt, IP}}}).
 
+validate_server_spec_list(_Opt, [], Acc) ->
+    lists:reverse(Acc);
+    
+validate_server_spec_list(Opt, [{NAS_Type, {IP, Port, Secret}} = H | T], Acc) 
```

Comment:
please avoid unedrscores in variable names
```suggestion
validate_server_spec_list(Opt, [{Type, {IP, Port, Secret}} = H | T], Acc) 
```

Reply:
done